e2e_test/js-tests/test_viem_tx.mjs

```diff
@@ -271,27 +309,38 @@ describe("viem send tx", () => {
 		}
 	}).timeout(10_000);
 
-	it("send fee currency tx with just high enough gas price", async () => {
+	it("send fee currency tx with just high enough gas price mine", async () => {
+		const rate = await getRate(process.env.FEE_CURRENCY);
 		const block = await publicClient.getBlock({});
-		// Actually, the base fee will be a bit lower next block, since our blocks
-		// are always mostly empty. But the difference will be much less than the
-		// exchange rate of 2, so the test will still check that we take the fee
-		// currency into account everywhere.
-		const maxFeePerGas = block.baseFeePerGas / 2n;
+		const maxFeePerGas = rate.toFeeCurrency(block.baseFeePerGas)+2n;
 		const request = await walletClient.prepareTransactionRequest({
 			account,
 			to: "0x00000000000000000000000000000000DeaDBeef",
 			value: 2,
-			gas: 90000,
-			feeCurrency: process.env.FEE_CURRENCY2,
-			maxFeePerGas,
-			maxPriorityFeePerGas: 1n,
+			gas: 171000,
+			feeCurrency: process.env.FEE_CURRENCY,
+			maxFeePerGas: maxFeePerGas,
+			maxPriorityFeePerGas: 2n,
```

Comment:
Hey @karlb, I'm thinking probably my modifications here don't make sense because it seems that it is important for this test that the fee per gas for the fee currency than the native currency, and I've modified it to use USDC which is the other way around.

Reply:
Yes, it seems that the main point of this is that the fee should be too low for native currency, but if the node converts it then it is sufficient.

Reply:
I will leave as is, with a comment that we may want to change this if the exchange rate changes significantly.